sys/checksum/doc.txt

```diff
@@ -10,4 +11,24 @@
  * @defgroup    sys_checksum Checksum
  * @ingroup     sys
  * @brief       Checksum function libraries
+ *
+ * This module provides a number of checksum functions. Most notably is the
+ * @ref sys_checksum_crc16_ccitt and the @ref sys_checksum_ucrc16 modules which
+ * provide support for the CRC16 checksum.
+ *
+ * @ref sys_checksum_crc16_ccitt only provides an implementation of the CCITT
+ * flavor of CRC16 (polynomial @$ x^{16} + x^{12} + x^{5} + 1 @$) for big-endian
+ * numbers with starting seed `0x1d0f` (though others can be provided), while
+ * @ref sys_checksum_ucrc16 is more generalized, since it takes the
+ * hexadecimal representation as a parameter and provides functions and
```

Comment:
reading this again after a while I think it makes sense to specify: the hex representation of what?